sentry/sentry.conf.example.py

```diff
@@ -314,9 +314,23 @@ def get_internal_network():
             "organizations:continuous-profiling",
             "organizations:continuous-profiling-stats",
         )
+        # Uptime related flags
+        + (
+            "organizations:uptime-create-issues",
+            # TODO(epurkhiser): We can remove remove these in 25.8.0 since
+            # we'll have released this issue group type
+            # (https://github.com/getsentry/sentry/pull/94827)
+            "organizations:issue-uptime-domain-failure-visible",
+            "organizations:issue-uptime-domain-failure-ingest",
+            "organizations:issue-uptime-domain-failure-post-process-group",
+        )
```

Comment:
Shouldn't we need `organizations:uptime` here?

https://github.com/getsentry/sentry/blob/cce6829b019961bfb233ed27eb4f303ce0c8fd72/src/sentry/features/permanent.py#L109-L110

I know that :point_up: defaults to `True`, yet we have `errors-only` mode in which shouldn't expose the uptime component.